extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientProcessor.java

```diff
@@ -313,4 +315,13 @@ HealthBuildItem addHealthCheck(MongoClientBuildTimeConfig buildTimeConfig) {
         return new HealthBuildItem("io.quarkus.mongodb.health.MongoHealthCheck",
                 buildTimeConfig.healthEnabled, "mongodb");
     }
+
+    @BuildStep
+    void produceMetricsEnabledProperty(BuildProducer<SystemPropertyBuildItem> producer,
+            MongoClientBuildTimeConfig buildTimeConfig, Capabilities capabilities) {
+
+        if (buildTimeConfig.metricsEnabled && capabilities.isCapabilityPresent(Capabilities.METRICS)) {
+            producer.produce(new SystemPropertyBuildItem("mongodb.metrics.enabled", "true"));
```

Comment:
What is this for? Is it something that the Mongo client library recognizes? I haven't been able to find what it is used for at runtime.